package day13;

public class Test147 {

	static void func() {
		Exception e =new Exception();
//			throw e;
	
			
	}
	public static void main(String[] args) {
		try {
			func(); // 메서드를 호출해 사용하는 쪽엥서 try -catch로 예외처리
		}catch(Exception e) {
			   
		}
	}
			
}


